osf/permissions.py

```diff
@@ -0,0 +1 @@
+INSTITUTION_ADMIN = 'institution_admin'
```

Comment:
Why is this defined in a new file as opposed to osf/utils/permissions.py?

Reply:
good point I just missed it.

Reply:
Drive by: @Johnetordoff, here is the PR with the permissions for institutional users. https://github.com/CenterForOpenScience/osf.io/pull/9297/files

Reply:
Ah Ok everyone on the same page, pull and work off this #9297 for permissions.